### PR TITLE
adds the t-26, and reprices the autosniper

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
adds the t-26, which is basically the old m42a, you lase a beno to deal more damage. Normal rounds only.
Reprices autosniper to 40 points.

## Why It's Good For The Game
autosniper is overpriced, and the antimaterial rifle is a neat idea

## Changelog
:cl:
add: Added new antimaterial rifle to marines, 120 points. Functions like the old m42a scout sniper
balance: autosniper repriced to 40
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -288. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Swagile** [comment]
Ive said this before on your main PR and ill say it again.

You do not want to nerf autosniper price. It is only balanced around the fact that you can't spam it.

Being able to buy tons of these will just bring back TL bolt meta except worse; you can't acid flares to stop this, and on top of it, autosniper can be used to push out for its NVG qualities, making it INSANELY good.

For reference, the tac binos is 30 points, so you'd basically have no real reason to buy tac binos outside of maybe OBing once in a while or railgun.

**Comment by SkeletalElite** [comment]
It's spelled anti-materiel rifle. It's a different word, not just a different spelling.

File: code/modules/codex/entries/guns_codex.dm [hunk 1]
```diff
@@ -181,7 +181,7 @@
 		It should be noted that the user will need to be aware and at the ready before discharging them."
 
 /datum/codex_entry/sniper_rifle
-	associated_paths = list(/obj/item/weapon/gun/rifle/sniper/M42A)
+	associated_paths = list(/obj/item/weapon/gun/rifle/sniper/antimaterial)
 	lore_text = "A rather strange gun in the TGMC's arsenal. The M42A \"Express\" originally was born out of it's younger brother the M42. Made by the same \
 	company who eventually went on to design the M56 smartgun system. Which the M42As specialized scope eventually adopted a modified IFF system similar to it's cousin the smartgun. <br><br>\
 	It was at first marketed to PMCs and civilians as an expensive accurate long range rifle but it failed due to the lack of need for such a thing for PMCs and the wide variety of options \
```
Comment (SkeletalElite) [inline code/modules/codex/entries/guns_codex.dm:187]: Codex entry still refers to it by the m42a

**Comment by revelation-8468** [comment]
>120 points

lol wut

**Comment by SpaceLoveSs13** [comment]
Doesn't T-26 already exist in the code. It's the Spec grenade launcher that can be obtained from grenediers kit.

**Comment by Pariah919** [comment]
that's not an item you can get anymore anyway

Its just the generic gl now

File: code/game/objects/items/portable_vendor.dm [hunk 2]
```diff
@@ -278,7 +278,7 @@ A breacher kit with the least firepower, has a high capacity SMG and a machete f
 	listed_products = list(
 		list("SPECIALIST KITS", 0, null, null, null),
 		list("Demolitionist (RPG, T-19 MP)", 1, /obj/item/storage/box/squadmarine/demolitionist, "white", "A kit specializing in explosives."),
-		list("Sniper (M42A Scoped Rifle, 88 Mod 4 Pistol)", 1, /obj/item/storage/box/squadmarine/sniper, "white", "A kit specializing in long-ranged dispatch."),
+		list("Sniper (antimaterial Scoped Rifle, 88 Mod 4 Pistol)", 1, /obj/item/storage/box/squadmarine/sniper, "white", "A kit specializing in long-ranged dispatch."),
```
Comment (TiviPlus) [inline code/game/objects/items/portable_vendor.dm:281]: Capitilizing
```suggestion
		list("Sniper (Antimaterial Scoped Rifle, 88 Mod 4 Pistol)", 1, /obj/item/storage/box/squadmarine/sniper, "white", "A kit specializing in long-ranged dispatch."),
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-306). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**